CRM/Event/Page/ManageEvent.php

```diff
@@ -482,7 +482,7 @@ public function whereClause(&$params, $sortBy = TRUE, $force) {
       }
       else {
         $curDate = date('YmdHis');
-        $clauses[5] = "(end_date >= {$curDate} OR end_date IS NULL)";
+        $clauses[] = "(end_date >= {$curDate} OR end_date IS NULL)";
       }
     }
     else {
```

Comment:
So if I understand correctly we do not need to filter in the Manage Events page since the filtering takes place in CRM_Event_BAO_Event::checkPermissions() ?